src/main.ts

```diff
@@ -310,13 +328,24 @@ export class Purchases {
       `Presenting purchase form for package ${rcPackage.identifier}`,
     );
 
-    return new Promise((resolve, reject) => {
+    return new Promise(async (resolve, reject) => {
+      if (this._brandingInfo === null) {
```

Comment:
I thought to add this as a fallback, in case there is an issue downloading initially